--- tmp-pyqt4-i686-w64-mingw32.static/PyQt4_gpl_win-4.12.1/configure-ng.py	2017-12-22 18:32:39.878059247 +1100
+++ PyQt4_gpl_win-4.12.1/configure-ng.py	2017-12-22 18:41:42.367554512 +1100
@@ -2403,7 +2403,7 @@
             spec = 'macx-g++'
 
     if 'g++' in spec or 'clang' in spec:
-        pro_lines.append('QMAKE_CXXFLAGS += -fno-exceptions')
+        pro_lines.append('QMAKE_CXXFLAGS += -fno-exceptions -fpermissive')
 
     # This optimisation could apply to other platforms.
     if 'linux' in spec and not target_config.static:
