diff -u -r qwtplot3d/include/qwt3d_openglhelper.h qwtplot3d1/include/qwt3d_openglhelper.h
--- qwtplot3d/include/qwt3d_openglhelper.h	2005-07-19 17:40:28.000000000 +1000
+++ qwtplot3d1/include/qwt3d_openglhelper.h	2019-03-05 09:44:15.533353191 +1100
@@ -7,6 +7,7 @@
 #else
 #include <QtOpenGL/qgl.h>
 #endif
+#include <GL/glu.h>
 
 namespace Qwt3D
 {
diff -u -r qwtplot3d/qwtplot3d.pro qwtplot3d1/qwtplot3d.pro
--- qwtplot3d/qwtplot3d.pro	2007-05-20 17:51:50.000000000 +1000
+++ qwtplot3d1/qwtplot3d.pro	2019-03-05 10:05:03.598014448 +1100
@@ -3,7 +3,7 @@
 
 TARGET            = qwtplot3d
 TEMPLATE          = lib
-CONFIG           += qt warn_on opengl thread zlib debug
+CONFIG           += qt warn_on opengl thread zlib release static
 MOC_DIR           = tmp
 OBJECTS_DIR       = tmp
 INCLUDEPATH       = include
@@ -12,8 +12,7 @@
 #DESTDIR      			= ../../../lib
 QT += opengl
 
-win32:TEMPLATE    = vclib
-win32:CONFIG     += dll exceptions
+win32:CONFIG     += exceptions
 win32:dll:DEFINES    += QT_DLL QWT3D_DLL QWT3D_MAKEDLL
 win32:QMAKE_CXXFLAGS     += $$QMAKE_CFLAGS_STL
 
@@ -92,4 +91,4 @@
   DEFINES += GL2PS_HAVE_ZLIB
   win32:LIBS += zlib.lib
 	unix:LIBS  += -lz
-}
\ No newline at end of file
+}
